src/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { assignAll } from 'redux-act';
+import { BrowserRouter, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import * as actions from './actions';
+import createStoreWithMiddleware from './store';
+import reducers from './reducers';
+
+const store = createStoreWithMiddleware(reducers);
+assignAll(actions, store)
```

Comment:
I checked it doesn't seem like actions bound with `assignAll` can be written using the thunk syntax we typically use for API requests. I don't think we need to drop or change this now but keeping it will eventually require segmenting actions into files based on

1. whether or not they are thunks
2. whether they accept multiple arguments or need to be curried or before dispatch
3. whether they might need internal access to `getState` to check on some value in the store

...which adds back some of the complexity we're trying to circumvent with the `assignAll` syntax.

I think I generally prefer using `mapDispatchToProps` and `bindActionCreators` -- https://redux.js.org/api/bindactioncreators -- to `assignAll` syntax, but it may be that this application's level of complexity works with `assignAll`.

Anyway: fine to leave as is, just thinking through the implications.

Reply:
Thanks for the overview. I prefer `assignAll` for its simplicity, but realize it might not be able to handle everything we need. It worked for Urban Water Revealed, but that was a fairly simple app. I'll keep this for now, but open to changes when we need them.